packages/next/client/image.tsx

```diff
@@ -291,6 +320,17 @@ export default function Image({
   const heightInt = getInt(height)
   const qualityInt = getInt(quality)
 
+  const MIN_IMG_SIZE_FOR_PLACEHOLDER = 100
```

Comment:
We may want to make this a configurable value later.

Reply:
Also since this is being compared against the product of width and height, 100 seems really low. Even a 10x10 icon would hit it. I think something around 5000 would be better?